runtime/lua/vim/tshighlighter.lua

```diff
@@ -25,6 +25,7 @@ TSHighlighter.hl_map = {
 function TSHighlighter.new(query, bufnr, ft)
   local self = setmetatable({}, TSHighlighter)
   self.parser = vim.treesitter.get_parser(bufnr, ft, function(...) self:on_change(...) end)
+  self._dummy = vim.treesitter.parse_query(self.parser.lang, "(_) @dummy")
```

Comment:
Is there really no way to iterate over matches without doing this? I do not like the look of the hack :laughing: